main.py

```diff
@@ -1,13 +1,45 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, HTTPException
 import uvicorn
-
+from pydantic import BaseModel
 from router import router
+from typing import List
 
 
 app = FastAPI()
 
 app.include_router(router)
 
 
+class User(BaseModel):
+    first_name: str
+    last_name: str
+    age: int
+
+
+user_table = [
+    User(first_name="user1", last_name="last_name1", age=2135),
+    User(first_name="user2", last_name="last_name1", age=55),
+    User(first_name="user3", last_name="lastname_666", age=20),
+]
+
+
+@app.get("/all", response_model=List[User])
+async def get_all_users():
```

Comment:
проставь тут возвращаемый тип, аналогично в других роутах
```suggestion
async def get_all_users() -> list[User]:
```